examples/cockroachdb/cockroachdb-blueprint.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: cockroachdb-blueprint
+  namespace: kanister
```

Comment:
```suggestion
```
Let's not add the namespace to the blueprint here. We can explicitly provide the namespace while creating the blueprint.